components/brave_wallet/browser/fil_tx_manager.cc

```diff
@@ -224,6 +228,48 @@ FilTxStateManager* FilTxManager::GetFilTxStateManager() {
   return static_cast<FilTxStateManager*>(tx_state_manager_.get());
 }
 
-void FilTxManager::UpdatePendingTransactions() {}
+void FilTxManager::UpdatePendingTransactions() {
+  auto pending_transactions = tx_state_manager_->GetTransactionsByStatus(
+      mojom::TransactionStatus::Submitted, absl::nullopt);
+  for (const auto& pending_transaction : pending_transactions) {
+    auto cid = pending_transaction->tx_hash();
+    uint64_t seconds =
+        (base::Time::Now() - pending_transaction->submitted_time()).InSeconds();
+    uint64_t depth = seconds == 0 ? 1 : seconds;
```

Comment:
I'm not sure I understand what's the logic of depth here, could you point me to some documentation of this?

Reply:
`StateSearchMsgLimited` looks back up to limit epochs in the chain for a message, and returns its receipt and the tipset where it was executed. There is no documentation to clarify this parameter unfortunately, guys from Filecoin slack advised to pass amount of blocks based on time needed for a block to be mined, they said it usually 10-30 seconds, but experimentally I found that looking through 1 block per second work more stable.

Reply:
Please document this in the code, thanks.
Also, I think limit_epochs is more suitable than depth.

Reply:
done